test/libsolidity/SemanticTest.cpp

```diff
@@ -264,6 +264,19 @@ vector<string> SemanticTest::eventSideEffectHook(FunctionCall const&) const
 		sideEffect << joinHumanReadable(eventStrings);
 		sideEffects.emplace_back(sideEffect.str());
 	}
+	if(sideEffects.size() > 0){
+		for(long unsigned int i = 0; i < sideEffects.size() - 1; i++)
+		{
+			if(i == 0)
```

Comment:
Wouldn't it make more sense to set this above the loop, since you're already checking that `sideEffects` is not empty, so the `sideEffects[0]` will always be present. This will allow you to avoid branching in the loop.